NodesConstraints.Core/NodesConstraints.cs

```diff
@@ -159,70 +177,98 @@ public void UpdateDebugLines()
                 _debugLine.Draw();
             }
 
-            public Vector3 GetMirroredPosition()
```

Comment:
Are all of the mirrored methods removed on purpose?

Reply:
Yes. When I added them at first it was because the mirrored constraint needed to use the parents change from their original transform to be able to mirror it, while the normal contraints could continue to use their original logic (parent transform + offset).

With these new features, the normal contraints need to use this same type of logic. So since the mirror version don't need to do anything special anymore, I just removed them to simplify it a little.
- normal and mirror position is just adding or subtracting the movement
- normal and mirror rotation is just inverting the change or not
- normal and mirror scaling is either a positive or negative power